models/stg_sage_intacct__gl_account.sql

```diff
@@ -60,3 +60,4 @@ final as (
 )
 
 select * from final
+where _fivetran_deleted = false
```

Comment:
Same comment as before, we should apply a safer filter here
```suggestion
where not coalesce(_fivetran_deleted, false)
```